src/Raven.Server/Utils/Cli/RavenCli.cs

```diff
@@ -419,10 +417,8 @@ private static bool CommandTopThreads(List<string> args, RavenCli cli)
 
             LoggingSource.Instance.DisableConsoleLogging();
             var prevLogMode = LoggingSource.Instance.LogMode;
-            SetupLogMode(LogMode.None, cli._server.Configuration.Logs);
-            Program.WriteThreadsInfoAndWaitForEsc(cli._server, maxTopThreads, updateIntervalInMs, cpuUsageThreshold);
             SetupLogMode(prevLogMode, cli._server.Configuration.Logs);
```

Comment:
why we want to setup prev log mode before doing any of the operations? we didn't change it after all?